cli/tsc/README.md

```diff
@@ -10,13 +10,13 @@ currently (unfortunately) have a rather manual process for upgrading TypeScript.
 It works like this currently:
 
 1. Checkout denoland/TypeScript repo in a separate directory.
-1. Add Microsoft/TypeScript as a remote and fetch its latest tags
-1. Checkout a new branch based on this tag.
-1. Cherry pick the custom commit we made in a previous release to the new one.
-1. This commit has a "deno.ts" file in it. Read the instructions in it.
-1. Copy typescript.js into Deno repo.
-1. Copy d.ts files into dts directory.
-1. Review the copied files, removing and reverting what's necessary
+2. Add Microsoft/TypeScript as a remote and fetch its latest tags
+3. Checkout a new branch based on this tag.
+4. Cherry-pick the custom commit we made in a previous release to the new one.
+5. This commit has a "deno.ts" file in it. Read the instructions in it.
+6. Copy typescript.js into Deno repo.
+7. Copy d.ts files into dts directory.
+8. Review the copied files, removing and reverting what's necessary
```

Comment:
It being all `1`s is a way to have an automatically numbered list https://stackoverflow.com/a/12145270/188246